google-home-community.groovy

```diff
@@ -2591,6 +2625,14 @@ private traitFromSettings_HumiditySetting(traitName) {
     return humidityTrait
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private traitFromSettings_Locator(traitName) {
+    return [
+        locatorCommand:   settings."${traitName}.locatorCommand",
+        commands:         ["Find my", "Where is"]
```

Comment:
These `commands` are used for multifactor.  The entries here should match the strings passed to `checkMfa` in this trait's command handlers.  IMO the only command should probably be `"Locate"`, since that's the only action you can take for this trait.